js_modules/dagster-ui/packages/ui-core/src/owners/DefinitionOwners.tsx

```diff
@@ -0,0 +1,42 @@
+import {Box, Tag} from '@dagster-io/ui-components';
+import {UserDisplay} from 'shared/runs/UserDisplay.oss';
+
+import styles from './css/DefinitionOwners.module.css';
+
+export interface UserOwner {
+  __typename: 'UserDefinitionOwner' | 'UserAssetOwner';
+  email: string;
+}
+
+export interface TeamOwner {
+  __typename: 'TeamDefinitionOwner' | 'TeamAssetOwner';
+  team: string;
+}
+
+export type Owner = UserOwner | TeamOwner;
+
+export interface DefinitionOwnersProps {
+  owners: Owner[];
+}
+
+export const DefinitionOwners = ({owners}: DefinitionOwnersProps) => {
+  if (!owners || owners.length === 0) {
+    return null;
+  }
+
+  return (
+    <Box flex={{gap: 4, wrap: 'wrap'}}>
+      {owners.map((owner, idx) =>
+        owner.__typename === 'UserDefinitionOwner' || owner.__typename === 'UserAssetOwner' ? (
+          <div key={idx} className={styles.user_display}>
+            <UserDisplay email={owner.email} size="very-small" />
+          </div>
```

Comment:
Thanks for the screenshot! Could this use `BaseTag`? It seems a little excessive to reimplement something that looks like a `Tag`.